server/app/services/question/types/QuestionDefinitionConfig.java

```diff
@@ -49,16 +53,12 @@ public interface RequiredDescription {
   }
 
```

Comment:
Previously, `RequiredQuestionText` was a clever mechanism to ensure that we set the question help text at compile time. Since `QuestionDefinitionBuilder` has many clients that don't set this, we should remove it as it doesn't match the business logic.